demo/src/screens/componentScreens/TextFieldScreen.tsx

```diff
@@ -15,9 +15,10 @@ import {
   SegmentedControl,
   Icon
 } from 'react-native-ui-lib';
+const {loadDemoConfigurations} = require('../../../src/configurations.js');
 const {KeyboardAwareInsetsView} = Keyboard;
 const priceFormatter = Intl.NumberFormat('en-US');
-
+loadDemoConfigurations();
```

Comment:
I'm using `loadDemoConfigurations` because in the `TextFieldScreen` we are using Assets after loading them under group `demo` in this case, getting an error while building the snapshot/running the test when not running `loadDemoConfigurations` .